packages/scripts/README.md

```diff
@@ -207,8 +213,38 @@ _Example:_
 ```
 
 This is how you execute those scripts using the presented setup:
+
 * `npm run test:unit` - runs all unit tests.
 * `npm run test:unit:help` - prints all available options to configure unit tests runner.
 * `npm run test:unit:watch` - runs all unit tests in the watch mode.
 
+## Webpack config
+
+The `build` and `start` commands use [Webpack](https://webpack.js.org/) behind the scenes. Webpack is a tool that helps you transform your code into something else. For example: it can take code written in ESNext and output ES5 compatible code that is minified for production.
+
+### Default Webpack config
+
+`@wordpress/scripts` bundles the default Webpack config used as a base by the WordPress editor. These are the defaults:
+
+* [Entry](https://webpack.js.org/configuration/entry-context/#entry): `src/index.js`
+* [Output](https://webpack.js.org/configuration/output): `build/index.js`
+* [Externals](https://webpack.js.org/configuration/externals). These are the transformations done to imports:
+
+Package | Input syntax | Output
+--- | --- | ---
+React | `import x from React;` | `var x = this.wp.react.x;`
+ReactDOM | `import x from ReactDOM;` | `var x = this.wp.react-dom.x;`
+moment | `import x from moment;` | `var x = this.wp.moment.x;`
+jQuery | `import x from jQuery;` | `var x = this.wp.jQuery.x;`
```

Comment:
I think there shound't be `wp` namespace used for non-WordPress libraries:

![screen shot 2019-03-04 at 11 02 34](https://user-images.githubusercontent.com/699132/53725821-09e6f880-3e6d-11e9-90b1-c455e60182fc.png)

* `this.React`
* `this.ReactDOM`
* `this.moment`
* `this.jQuery`
* `this.lodash` for both versions of lodash - we alias it to ensure it is bundled only once.

Reply:
Fixed at 9f3f2d8